virt/release_notes/virt-4-21-release-notes.adoc

```diff
@@ -76,6 +76,14 @@ The SVVP certification applies to:
 [id="virt-4-21-deprecated_{context}"]
 == Deprecated features
 
+The `HotplugVolume` feature gate is deprecated::
+
+The `HotplugVolume` feature gate, which allows you to add storage without restarting your VM, is deprecated and will be removed in a future release. This feature gate will be replaced by `DeclarativeHotplugVolumes`.
+
+NOTE: `DeclarativeHotplugVolumes` does not support ephemeral hotplug volumes. Ephemeral hotplug volumes are volumes that are hotplugged to a VMI and do not persist in the owner VM. Existing ephemeral hotplug volumes are automatically detached after you switch to `DeclarativeHotplugVolumes`.
```

Comment:
```suggestion
[NOTE]
====
`DeclarativeHotplugVolumes` does not support ephemeral volumes. Ephemeral volumes are hot plugged to a VMI and do not persist in the owner VM. Existing ephemeral volumes that are hot plugged are automatically detached after you switch to `DeclarativeHotplugVolumes`.
====
```

s/hot plug/hotplug based on the [Red Hat Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#hot-plug)